buildsettings.py

```diff
@@ -22,7 +22,7 @@
 
     # these settings should not be touched unless intel url(s) changed
     'url_intel_base': 'https://intel.ingress.com/',
-    'match': 'https://intel.ingress.com/*',
+    'match': '*://intel.ingress.com/*',
```

Comment:
If I understand correctly - we need this change only for main script.

Reply:
Sure. But ATM there are no separated main & plugin settings and changing all doesn't hurt.

Reply:
> changing all doesn't hurt.

Well, we know that standard plugins serve as samples and templates for 3rd-party, that's why I believe that it'd better to have single form - https.

Also I'm not sure that we should put that workaround into core. Maybe we should resurrect "force http" plugin instead.

And see https://github.com/IITC-CE/ingress-intel-total-conversion/commit/fe2d240eb77bf7f05de45a73c93cd33e29ff2073 - there was separate routine in mobile app, which also may be needed.

Reply:
> Maybe we should resurrect "force http" plugin instead.

This would only help these people who might know they run into problems. And they know how to fix it.

> And see [fe2d240](https://github.com/IITC-CE/ingress-intel-total-conversion/commit/fe2d240eb77bf7f05de45a73c93cd33e29ff2073) - previously there was separate routine in mobile app,

IDK how mobile reacts on http links. I thought it would be able to handle all registered web-links.

Reply:
> Well, we know that standard plugins serve as samples and templates for 3rd-party, that's why I believe that it'd better to have single form - https.

why is it BAD to support http, too?

Reply:
> why is it BAD to support http too? Niantic supports both.

For example because:
1) it would lead us to old "bug" with inconsistent plugin data, because localStorage is not shared between different protocols.
2) http is vulnerable, and there is no single reason to prefer it in IITC
3) there is pretty big chance tomorrow Niantic will forbid http again.

So we just shouldn't encourage "http" because nobody gain from it.

But anyway I agree that we should reimplement http->https redirection on our side.
Question is whether put it into core or plugin.

Reply:
as said: as plugin you won't help anybody.

my PR was for people not knowing they use the wrong page.